### ISSUE TITLE
Current state of cathook

### ISSUE THREAD

**Issue by raphael-costa** [nullworks/cathook#801]
My apologies for starting an unnecessary thread but I wanted to ask "What's the current state of cathook?"

Yes, I want to cheat but I don't want to troll either, I don't want to hop in into a server as Sniper and start shooting invisible spies and get a Godlike killstreak 5 minutes into the game. I want to play something like Scout and get some free shots once in a while that I would miss or play Pyro and "get" those difficult airblasts I can't seem to time right. Basically, I want a crutch, I want to do better than what I really can with my current skill-level, I want to hit those airshots as Soldier a few times even if it's with a cheat.

Will I get auto-banned for doing this?

And most importantly, if I create an alternative steam account do I need to do anything excessive to avoid being banned in my main account? I was thinking about just installing TF2 as a new user, do I need to take extra steps to ensure my main account doesn't get banned too?

**Comment by BenCat07** [comment]
Alright, i have a few things for you.

1. Being banned while using cathook is near impossible, you basically have to get steams attention, which is near impossible.
2. In the Case you want to really annoy Steam for a really long reason, to a void a ban on your main You can use https://gitlab.com/nullworks/just-disable-vac-tf, you need to manually compile and preload it into steam though, i will Make this process Easier when the time comes.
3. VAC does basically nothing as of right now. All it can do is
- Read attached Libraries (we randomize our Library name)
- Read attached debuggers ( Vac disabler Module deals with it, but it's not needed as valve reads the attached debugger **once like every 30 minutes** and we only attach for a few seconds at most)
- Construct a "HWID" based on your Device structure (vac disable deals with that once again, also only used for manual bans)


Also here i have some Bans Valve attempted for you:
- Ban by Library Name
- Ban if "cat-bot" in linux username
- Manually ban a person [by their "HWID"] (by the way, a reinstall of linux may even change it, or just changing a part of hardware).


In conclusion, you don't have any worries besides valve suddenly employing competent employees that can actually do something.

**Comment by raphael-costa** [comment]
So if I don't use just-disable-vac-tf I may get banned on my main due to hardware detection, right?

And I heard if you use a free cheat in a public server you get banned in hours/minutes, was that a flat lie? Or was it just that LMAOBOX gimps its free version on purpose?

**Comment by BenCat07** [comment]
Use just-disable-vac-tf module to be 100% safe on your main, ot just don't use your main on linux, that also works. The free cheat ban thing only applies to Windows, Not Linux

**Comment by BenCat07** [comment]
I explained everything vac can do above, Detecting Cheats properly is not one of those things

**Comment by raphael-costa** [comment]
I see, thanks! I'll be careful though. :+1:

**Comment by raphael-costa** [comment]
Wait, I have another question. I looked it up and it seems some time ago there was a huge surge of catbots in public servers, people queued like 12 bots at once to get a lobby full of bots but then a mass VAC banwave came and they disappeared, isn't this a problem?

**Comment by BenCat07** [comment]
Catbots still exist, and it wasn't a real banwave, they manually found the owner because he dragged so much attention to hilself and Banned him by HWID, which also hit the catbots
